src/styles/main.scss

```diff
@@ -0,0 +1,68 @@
+$day-side: 100px;
+$padding: 10px;
+$days-names: 'mon' 1,
+  'tue' 2,
+  'wed' 3,
+  'thu' 4,
+  'fri' 5,
+  'sat' 6,
+  'sun' 7;
+
+body {
+  margin: 0;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  margin: auto;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  width: calc(#{$day-side} * 7 + (1px * 6));
+  padding: 0 #{$padding};
+
+  .calendar__day {
+    background-color: #eee;
+    box-sizing: border-box;
+    height: $day-side;
+    width: $day-side;
+    border: 1px solid black;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    color: bla;
```

Comment:
There is a typo in the color property value: 'bla'. It should be corrected to a valid color value, such as 'black' or a hex code like '#000'.